### DISCUSSION TITLE
download cancel (Nth)

### DISCUSSION THREAD

**Discussion by Combowombo** [qutebrowser/qutebrowser#8086] [General] [answered]
I looked through the documentation but still not clear on how to make the most of the "download cancel" (and other download commands). I know i can use download-cancel to cancel the most recent download in a array of downloads. I assume that if I wanted to cancel the 1st on the list i would put "download-cancel 1", or if I wanted to cancel the 2nd I would put "download-cancel 2", but qutebrowser gives me an error.

Am i right to assume that this feature is not available yet? Or is there something I'm missing?

TLDR: I want to be able to use hotkeys to cancel and earlier download, not just the most recent with "download-cancel"

**Comment by The-Compiler** [comment, marked as answer]
The description for `:download-cancel` says:

> Cancel the last/[count]th download.

You pass a count to a command by:

1) Prefixing it to a keybinding, like in vim - so since `:download-cancel` is bound to `ad` by default, e.g. `2ad` to cancel the second download
2) or, using the `:cmd-run-with-count` command, i.e. `:cmd-run-with-count 2 download-cancel`

**Comment by Combowombo** [comment]
awesome! I had a suspicion the feature was there but couldn't figure it out from reading documentation. Thanks for the fast reply!
